packages/inter-protocol/test/psm/setupPsm.js

```diff
@@ -61,10 +60,7 @@ export const setupPsmBootstrap = async (
   timer = buildManualTimer(console.log),
   farZoeKit,
 ) => {
-  if (!farZoeKit) {
-    farZoeKit = await setUpZoeForTest();
-  }
-  const { zoe } = farZoeKit;
+  const { zoe } = await setUpZoeForTest(farZoeKit);
```

Comment:
Having `setUpZoeForTest` accept an argument which if present is returned with no side effects seems a bit misleading to me. I think things would be more clear with the nullary signature:
```suggestion
  const { zoe } = await (farZoeKit || setUpZoeForTest(farZoeKit));
```

Reply:
done